src/commonMain/kotlin/fr/acinq/lightning/channel/Commitments.kt

```diff
@@ -99,7 +101,7 @@ sealed class LocalFundingStatus {
     abstract val signedTx: Transaction?
     abstract val txId: ByteVector32
 
-    data class UnconfirmedFundingTx(val sharedTx: SignedSharedTransaction, val fundingParams: InteractiveTxParams, val createdAt: Long) : LocalFundingStatus() {
+    data class UnconfirmedFundingTx(val sharedTx: SignedSharedTransaction, val fundingParams: InteractiveTxParams, val localCommitSig: CommitSig, val createdAt: Long) : LocalFundingStatus() {
```

Comment:
I understand why you put this here (need to resend at reconnection in the rbf case), but it's inconsistent with the `WaitForFundingSigned` case, and probably footgunny because this may not be in sync with the actual commitment in the zero-conf case.

I think we should persist the `InteractiveTxSigningSession` in the rbf (and splice) cases.

Reply:
Removed in https://github.com/ACINQ/lightning-kmp/pull/426/commits/46f9be9a8e46585432ba51f9b503e7eb22b4c3e7, we now re-compute our `commit_sig` when necessary, which is better.

Reply:
At first I thought there would be an opportunity for factorization between `RemoteCommit.sign` and `Commitment.sendCommit` but there is a chicken-and-egg issue.